lib/src/controllers/realtime.dart

```diff
@@ -156,7 +160,9 @@ class RealTimeController extends KuzzleController {
 
   void _renewSubscribe() async {
     for (final subs in _subscriptionsCache.values) {
-      for (final sub in subs) {
+       final List<Subscription> _localSubs = List<Subscription>.from(subs);
+        subs.clear();
```

Comment:
Can you fix the indentation here as well please? I can't do it from Github since the rest of the `subscribe` method is not in the diff

Reply:
I have fixed the indentation.